android/gradle.properties

```diff
@@ -14,4 +14,4 @@ android.useAndroidX=true
 android.enableJetifier=false
 
 # remove if com.android.tools.build:gradle can be raised in /android/build.gradle
-android.suppressUnsupportedCompileSdk=33
+android.suppressUnsupportedCompileSdk=33,34
```

Comment:
Question: where is this property documented? I can't find anything on it. How did you guess it was a comma separated list?

Reply:
I was an error during the build like you see in this post: https://stackoverflow.com/questions/69911807/we-recommend-using-a-newer-android-gradle-plugin-to-use-compilesdkpreview-sv2

Once we merge the gradle update PR we can remove it